ggml.c

```diff
@@ -6343,7 +6343,7 @@ static struct ggml_tensor * ggml_cpy_impl(
     }
 
     // make a view of the destination
-    struct ggml_tensor * result = ggml_view_tensor(ctx, b);
+    struct ggml_tensor * result = b->op == GGML_OP_NONE ? b : ggml_view_tensor(ctx, b);
```

Comment:
This is to avoid creating a new leaf with `ggml_cpy(.., ggml_new_tensor(..))`.

Reply:
Yeah, this change to `ggml_cpy` is not going to work. Maybe we could add an `ggml_cont_reshape` or just `ggml_cont_4d` that does the same thing?